perf/uncore_libpfm_test.go

```diff
@@ -199,3 +185,37 @@ func TestUncoreParseEventName(t *testing.T) {
 	assert.Equal(t, "some_pmu", pmuPrefix)
 	assert.Equal(t, "some_event/first_slash/second_slash", eventName)
 }
+
+func TestCheckGroup(t *testing.T) {
```

Comment:
Consider using [table driven tests](https://github.com/golang/go/wiki/TableDrivenTests).

Reply:
I think that positive scenario (`nil` error returned) is missing.